workers/unity/Packages/com.improbable.gdk.core/SchemaObjectExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Improbable.WorkerCore;
+
+namespace Improbable.Gdk.Core
+{
+    public static class SchemaObjectExtensions
+    {
+        public static void AddEntityId(this SchemaObject obj, uint fieldId, EntityId entityId)
```

Comment:
Is it not going to get confusing to have these for schema objects but not for the connection?

Reply:
I'd rather not add the extension methods unless it's there's something that makes the conversion particularly irritating.

Reply:
What APIs do we expose directly on the connection where a user hits an `long` instead of an `EntityId`? 

The alternative is to special case EntityId **everywhere** in the codegen

Reply:
In a normal workflow

Reply:
Discussed offline - this is okay for now, we can revisit in the future when we do passes over the code generator